### ISSUE TITLE
River 2 Pro has fewer sensors and switches?

### ISSUE THREAD

**Issue by TommySharpNZ** [tolwi/hassio-ecoflow-cloud#350]
With the most recent versions the River 2 Pro seems to now have less sensors and switches then the 2 and 2 Max? Is this intentional???

**Comment by TommySharpNZ** [comment]
Any word on why this model has some missing sensors when compared with the other River 2 series?

**Comment by sauron3256** [comment]
That would be because it has less sensors as a unit. Unlike the other River products, for example, usb is always on (no switch to manage it) and it doesn't have any expandability options with extra batteries etc, so there are less entities to manage.

**Comment by TommySharpNZ** [comment]
Yes some recent changes to this integration caused it to lose the options for setting some of the "eco" stuff like the discharge level...

**Comment by TommySharpNZ** [comment]
Any news on when we might be able to get the Energy Management options for the River 2 Pro?
The Backup Reserve to specify when to charge from solar only etc…
